src/engraving/libmscore/segment.cpp

```diff
@@ -2620,4 +2626,49 @@ Fraction Segment::shortestChordRest() const
     }
     return shortest;
 }
+
+/* layoutGraceNotes()
+ * Needs to be called when creating the segment shape. Make sure it is called *last*,
+ * so that grace notes can be correctly padded against the segment shape.*/
+void Segment::layoutGraceNotes(int staffIdx)
```

Comment:
https://github.com/musescore/MuseScore/pull/11394/files#r866118008

the same story about this one